recipes/tbb/config.yml

```diff
@@ -9,3 +9,9 @@ versions:
     folder: all
   "2020.3":
     folder: all
+  "2021.1.1":
+    folder: oneTBB
+  "2021.2.0":
+    folder: oneTBB
+  "2021.3.0":
+    folder: oneTBB
```

Comment:
It would be more efficient (for CI) to add only one version to begin with, since it's like a new recipe.